### PR TITLE
Feature/basic navigation

### LINKED ISSUE #6: Navigation
## Value Proposition
As a tutee[^1], I want to be able to quickly return to my tutorial collection on the library page at any time and from any page to select another tutorial.

## Description
capstone-userstory-05-2

## Acceptance Criteria
- [x] shows a navbar at the bottom of every page
- [x] navbar displays a book icon and the text "Library", linking to the `LibraryPage`
- [x] on `LibraryPage`, the icon is highlighted

## Tasks
- [x] create a `Navbar` component including an book icon and text "Library"
- [x] integrate Routing
- [x] create Routing path 

[^1]: a person who is tutored

### PR DESCRIPTION
**Pull request by kai-franke**
In this PR I integrated and styled the basic navigation with a first menu item. I would be very happy about feedback. :)

**PREVIEW:** https://capstone-project-git-feature-basic-navigation-kai-franke.vercel.app/tutorials

### CODE REVIEW

File: pages/tutorials/[slug].js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default function Tutorial({ name, steps, id, slug }) {
 
       <ButtonContainer>
         {currentStep === 0 ? (
-          <Link href="/tutorials">
+          <Link href="/tutorials" passHref>
```
Comment (kai-franke) [inline pages/tutorials/[slug].js:66]: I've added passHref here, although I'm not sure if that makes sense for a button.

File: components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+import { IconContext } from "react-icons";
+
+function Footer() {
+  const router = useRouter();
+  
+  return (
+    <footer>
+      <NavBar>
+        <Link href="/tutorials" passHref>
+          <NavItem isActive={router.pathname === "/tutorials"}>
```
Comment (CathrinTruchan) [inline components/Footer.js]: Just a small suggestion: You could deconstruct the pathname from useRouter then you don't need to write router.pathname very time ;)

**Review by CathrinTruchan — APPROVED** [review]
Looks great. Just one small comment.

File: components/Footer.js (hunk 2 above)
Comment (kai-franke) [reply components/Footer.js]: Very good idea! Thank you, @CathrinTruchan !

**Pull request merged**